### PR TITLE
Create INSTALL; update README; delete README-windows

### LINKED ISSUE #974: Documentation has become out of sync
The README.md is probably the best place to go for installation instructions.

The wiki refers to Node v0.10.33.  Should this page be removed or updated to point back to README.md?  README-windows.md can probably be removed.

As an enhancement it would be worthwhile including a table which lists the required versions of various software.

@chucklorenz, it would be good to get your thoughts on this.

### PR DESCRIPTION
**Pull request by chucklorenz**
Fixes #974 

Includes changes proposed earlier in this thread. 
Also new pages have been created on the wiki.

### CODE REVIEW

**Comment by brian-learningpool** [comment]
+1, nice job. :+1:

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+* [Grunt](http://gruntjs.com/)
+* [Adapt-CLI](https://github.com/adaptlearning/adapt-cli)
+* [MongoDB](http://docs.mongodb.org/manual/)
+
+The following are optional:
+* [FFmpeg](https://www.ffmpeg.org/index.html) is used to provide thumbnails for image and video assets.
+* [Robomongo](http://robomongo.org/) is not used by the authoring tool, but you might find it helpful in your work with MongoDB.
+
+> **Tips:**
+> + Windows users should run these commands in Git Bash if Git was installed using default settings. Otherwise run run the command prompt window as Administrator.
```
Comment (dancgray) [inline INSTALL.md]: Run is in twice "Otherwise run run the command".

**Comment by dancgray** [comment]
+1 Looks really good.

**Comment by dennis-learningpool** [comment]
+1 - fine work, as always

**Comment by chucklorenz** [comment]
Thanks for spotting 'run run' @dancgray.

**Pull request merged**